wandb_ng.egg-info/PKG-INFO

```diff
@@ -115,6 +115,6 @@ Classifier: Topic :: System :: Logging
 Classifier: Topic :: System :: Monitoring
 Requires-Python: >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*
 Description-Content-Type: text/markdown
+Provides-Extra: aws
 Provides-Extra: kubeflow
 Provides-Extra: gcp
-Provides-Extra: aws
```

Comment:
I was seeing this in my local environment too.  @raubitsj is this ordering non-deterministic?

Reply:
annoying,  probably different pip versions or something crazy,  We might not need this egg-info checked in like this.  It comes from tox doing pip installs i think